core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -586,12 +592,28 @@ private[ui] class JobPagedTable(
   override def row(jobTableRow: JobTableRowData): Seq[Node] = {
     val job = jobTableRow.jobData
 
+    val killLink = if (killEnabled) {
```

Comment:
Hm, does this work? there's no 'else' so this becomes an `Any` and I don't know what happens below when it's rendered as a string. Should it be an empty string otherwise?